instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/trace/SpanE2ETests.kt

```diff
@@ -47,11 +48,13 @@ class SpanE2ETests {
     fun trace_span_set_operation_name() {
         // this will also test the existence of the span itself
         val testMethodName = "trace_span_set_operation_name"
-        GlobalTracer.get()
-            .buildSpan("random")
+        val tracer = GlobalTracer.get()
+        val span = tracer.buildSpan("random")
             .start()
-            .setOperationName(testMethodName)
-            .finish()
+        measure(PERF_PREFIX + testMethodName) {
+            span.setOperationName(testMethodName)
+        }
+        span.finish()
```

Comment:
Same here and below ?

Reply:
no, we are measuring individual methods. probably I will write dedicated method just to measure `finish` alone.